### DISCUSSION TITLE
Unable to find the Passphrase for SSH KEY

### DISCUSSION THREAD

**Discussion by IgnazMohamed** [community/community#34097] [New to GitHub]
I have recently joined Github to study an online course,  I am unable find the Passphrase for the key  generated from MacOS. Do i have to generate a new  ssh key?  and if so how I could proceed further? 
<img width="584" alt="Screenshot 2022-09-25 at 02 37 21" src="https://user-images.githubusercontent.com/114281888/192119646-7ea1e8e6-b113-4533-b83f-352f6dbf0945.png">

**Comment by airtower-luna** [comment]
If you can't find the passphrase, then yes, creating a new key is your best option. The process is exactly the same as when you [set up an SSH key for the first time](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent). However you might want to delete the old key first, both from your account and on disk, to avoid any confusion.

**Reply by IgnazMohamed** [reply]
Thanks for your prompt reply.

**Reply by IgnazMohamed** [reply]
Could you please provide me the instructions how to delete the existing ssh key from my account and on disk?

**Reply by airtower-luna** [reply]
For your account, got to setting and look under "SSH and GPG keys". There should be a list of keys, with a "delete" button for each.

For a key on disk just delete the files: the one SSH mentions when asking for the passphrase is the private key, there should be a matching one with a `.pub` ending for the public key. Just double check if you have the right files, once you delete the key from disk you can't recover it (unless you have a backup). :warning:
